### PR TITLE
Corpus Viewer: Display the number of query matches

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Implements #659.

##### Description of changes
Add information on a number of query matches.
Clean up Info box.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/text/widgets/owcorpusviewer.py [hunk 1]
```diff
@@ -57,17 +57,15 @@ def __init__(self):
...
-        gui.label(info_box, self, 'Documents: %(n_documents)s')
-        gui.label(info_box, self, 'Preprocessed: %(is_preprocessed)s')
-        gui.label(info_box, self, '  ◦ Tokens: %(n_tokens)s')
-        gui.label(info_box, self, '  ◦ Types: %(n_types)s')
-        gui.label(info_box, self, 'POS tagged: %(is_pos_tagged)s')
-        gui.label(info_box, self, 'N-grams range: %(ngram_range)s')
+        gui.label(info_box, self, 'Tokens: %(n_tokens)s')
+        gui.label(info_box, self, 'Types: %(n_types)s')
         gui.label(info_box, self, 'Matching: %(n_matching)s')
+        gui.label(info_box, self, 'Matched queries: %(n_words)s')
```
Comment (ajdapretnar) [inline orangecontrib/text/widgets/owcorpusviewer.py:68]: "Matched queries" is bad, but I cannot think of a better label. Ideas?

Comment (PrimozGodec) [reply orangecontrib/text/widgets/owcorpusviewer.py:68]: Currently, I cannot get any better idea. My suggestion is maybe `matching objects`.

Comment (PrimozGodec) [reply orangecontrib/text/widgets/owcorpusviewer.py:68]: But I don't like it either

**Comment by PrimozGodec** [comment]
I fixed test that started to fail after last changes

**Comment by ajdapretnar** [comment]
@lanzagar I have fixed everything we discussed. The tests will fail due to the TweetProfiler server being migrated.

**Pull request merged**